src/boot/efi/meson.build

```diff
@@ -375,6 +375,7 @@ common_sources = files(
         'console.c',
         'devicetree.c',
         'disk.c',
+        'drivers.c',
```

Comment:
Why move this here? It's not used by the stub.

Reply:
Indirect dependency.
qemu_open() added to util.c uses reconnect() which is in drivers.c